app/controllers/shopping_carts_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ShoppingCartsController < ApplicationController
+  before_action :get_shopping_cart
+
+  def checkout
+
```

Comment:
Please display message something like "You can download the image from: " and the url